pandas/tests/test_expressions.py

```diff
@@ -170,16 +173,9 @@ def testit():
 
             op = getattr(operator, opname)
 
-            result = expr._can_use_numexpr(op, op_str, left, left, "evaluate")
```

Comment:
so this will just raise now? if so can you add a test

Reply:
Actually, it seems we just return False. So I could leave that as a test, but note that this is a file-internal helper function (leading underscore) that's within pandas never called with a DataFrame. So I would basically be testing something that is not expected to have any specific behaviour.